string name
int32 id
float64 prob
int32 right
int32 top
int32 left
int32 bottom
